instructure/canvas-ios
# Fix the issue of Tab bar covering out color-palette-chooser while annotating a file

refs: [MBL-18074](https://instructure.atlassian.net/browse/MBL-18074)
affects: Student
release note: A walk-around fix for the issue of Tab bar covering out color-palette-chooser while annotating a file

## What's changed

This is to perform as a work around for the issue, as it was confirmed by PSPDFKit developers.
I opened a ticket on their support system to report that. **Request #120402**

This PR change is basically built around hide/show the tabBar when the annotation toolbar is about to show/hide 

## Test Plan

- Navigate to a PDF file in sections where tab bar would presented, like in Assignments or Course Discussions.
- Tap on Annotate button to show Annotation toolbar.
- The tab bar should be hidden.
- Dismiss annotation toolbar or hit on back button.
- The tab bar should be shown again.

## Screenshots

**Before**

https://github.com/user-attachments/assets/cef34ac8-e509-4f9a-8af5-d6eec9370ddf

**After**

https://github.com/user-attachments/assets/621fdfc4-4862-49a4-aa09-a3d816d949d5

## Checklist



[MBL-18074]: https://instructure.atlassian.net/browse/MBL-18074?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
